### ISSUE TITLE
When a SortedDictionary is serialized, does Aqla keep the "Comparer" of it?

### ISSUE THREAD

**Issue by inethui** [AqlaSolutions/AqlaSerializer#59]
I noticed that when a SortedDictionary is serialized/deserialized, its keys are sorted by the natural order of the key's type. The "Comparer" assigned to the SortedDictionary is lost.

**Comment by AqlaSolutions** [comment]
You are right, the comparer is not stored. Unless the comparer is serializable too, it's even impossible to implement. If the deserialization creates a new instance of the sorted dictionary, it doesn't know which comparer to use.
